sdk/src/event_consumer_trait/event_consumer.rs

```diff
@@ -0,0 +1,35 @@
+use crate::event_consumer_trait::EventConsumerError;
+use crate::models::messages::PolledMessage;
+
+/// Trait for event consumer
+#[allow(dead_code)] // Clippy can't see that the trait is used
+#[trait_variant::make(EventConsumer: Send)]
+pub trait LocalEventConsumer {
```

Comment:
Maybe we could add this trait to `mod.rs` in `iggy_consumer_ext`, since it's used here? Higher cohesion or something :D